### ISSUE TITLE
Octave legend incorrectly detects all entries

### ISSUE THREAD

**Issue by okomarov** [matlab2tikz/matlab2tikz#607]
```
h = plot(magic(3));
legend(h(2), 'straight line')
```
![capture](https://cloud.githubusercontent.com/assets/3731173/7049712/9cf69f5e-de13-11e4-9b27-60198bcbad10.png)

**Comment by PeterPablo** [comment]
I think we have this plot in `ACID(68)`. Would you please adapt it to your code above?

**Comment by PeterPablo** [comment]
Note: Check output of `ACID(28)`.

**Comment by PeterPablo** [comment]
fixed by #653 
![image](https://cloud.githubusercontent.com/assets/4340267/7565683/3e676d94-f7f2-11e4-94a3-98ec1ea73ebe.png)

```
h = plot(magic(4),'LineWidth',3);
legend(h([2,4]), 'line 2','and line 4')
title('legend entries should be aligned left')
```
